master/usage/removing-nodes.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Removing Nodes
+---
+
+With each Calico node there is a Node entry in the datastore and Workload
+Endpoints are associated with that node.  As nodes are replaced (with
+differently named nodes) or decomissioned, the datastore will accumulate
+node entries and possibly workload entries.  Currently there is not an
+automatic reaping of these old Node and Endpoint entries so it is necessary
+to periodically clean up these old entries or ensure they are removed when
+nodes are decomissioned.  This page provides instruction of how to remove
+those entries.
+
+> **Note**: Removing a Node entry will also remove the Workload Endpoint
+entries associated with that Node.
+
+### Removing a single Calico Node entry
+
+See the example below for how to remove a node with the calicoctl command.
+
+**Caution**
+- Before running the following make sure you are certain that the node is no
+  longer in service as all workloads associated with the node will be removed
+  from the datastore.
+- Any configuration specific to the node will be removed.  This would be
+  configuration like node BGP peerings or custom Felix configs.
+
+```
+calicoctl delete node <nodeName>
+```
+
+### Removing multiple Calico Node entries
```

Comment:
Maybe change this to "Remove all Calico Node resources" and don't bother talking about editing the all_nodes file.  This makes the distinction cleaner and easier to warn people about.  

If someone wants to delete multiple it would probably still be easier to delete a single entry at a time by the time they've edited the file.